crates/shared/src/price_estimation/competition/mod.rs

```diff
@@ -129,6 +137,10 @@ impl<T: Send + Sync + 'static> CompetitionEstimator<T> {
             }
         }
 
+        if results.is_empty() {
```

Comment:
Can results really be empty given that we also push errors into them?

Reply:
You are definitely right :facepalm: thank you!